x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/fulltext/Match.java

```diff
@@ -54,21 +70,68 @@ public Match(
             name = "query",
             type = { "keyword", "text" },
             description = "Text you wish to find in the provided field."
-        ) Expression matchQuery
+        ) Expression matchQuery,
+        @Param(
+            optional = true,
+            name = "boost",
+            type = { "integer", "double" },
+            description = "Boost value for the query."
+        ) Expression boost,
+        @Param(optional = true, name = "boost", type = { "integer", "keyword" }, description = "Query fuzziness") Expression fuzziness
```

Comment:
> f they only want to specify fuzziness, what do they do about boost?

Set it to 1.0, being that the neutral boost. We can document it.

> Have we considered providing them as inputs in a different way? For example as a map(name-value pairs), so that users can specify the match option names explicitly?

I've used the current language constructs to implement this. There is already a [function named parameters proposal](https://docs.google.com/document/d/1kbJOc64Izdi3VkkeH6GH6dakNut5xfQd14UDDtMKGGI/edit?tab=t.0#heading=h.wocyg3dmamj). This is a broader conversation than the `match` function as it will impact the language.

> Minor comment, there is a typo in the fuzziness param name.

Thank you, fixed in e6ab030fed7423be234cda6becbe86aef50acb6f!